src/main/java/edu/wpi/first/pathweaver/MainController.java

```diff
@@ -337,9 +337,9 @@ private void buildPaths() {
       TankModifier tank = path.getTankModifier();
       Pathfinder.writeToCSV(new File(output, path.getPathNameNoExtension() + ".pf1.csv"),
           tank.getSourceTrajectory());
-      Pathfinder.writeToCSV(new File(output, path.getPathNameNoExtension() + ".left.pf1.csv"),
-          tank.getLeftTrajectory());
       Pathfinder.writeToCSV(new File(output, path.getPathNameNoExtension() + ".right.pf1.csv"),
```

Comment:
Should leave a comment here to explain the hack and why the left path goes to the right-side file and vice-versa.